src/store/modules/order/actions.ts

```diff
@@ -42,6 +42,25 @@ const actions: ActionTree<OrderState , RootState> ={
     return resp;
   },
 
+  async getOrderDetail( { dispatch }, payload ) {
+    let resp;
+
```

Comment:
We should check if the order is already in the list

Reply:
Added condition to check if the order exists or not.